src/main/java/practice/StreamPractice.java

```diff
@@ -1,80 +1,68 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
-    /**
-     * Given list of strings where each element contains 1+ numbers:
-     * input = {"5,30,100", "0,22,7", ...}
-     * return min integer value. One more thing - we're interested in even numbers.
-     * If there is no needed data throw RuntimeException with message
-     * "Can't get min value from list: < Here is our input 'numbers' >"
-     */
+
+    private static final CandidateValidator candidateValidator = new CandidateValidator();
+
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(e -> Arrays.stream(e.split(",")))
+                .filter(e -> Integer.parseInt(e) % 2 == 0)
+                .mapToInt(Integer::parseInt)
```

Comment:
and better to make `num % 2 == 0` a method like `isEven()` and reuse it in other places
```suggestion
                .mapToInt(Integer::parseInt)
                .filter(e -> e % 2 == 0)
```